backend/src/controllers/analysisController.js

```
@@ -0,0 +1,20 @@
+     * Handles the request to get holder analysis data.
+     */
+    static async getHolderAnalysis(req, res) {
+        try {
+            const analysisData = await AnalysisService.calculateHolderMetrics();
+            res.status(200).json(analysisData);
+        } catch (error) {
+            logger.log(`Error fetching holder analysis: ${error.message}`);
+            res.status(500).json({ message: 'Internal Server Error' });
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)